packages/odie-client/src/components/message/style.scss

```diff
@@ -110,15 +110,109 @@ $blueberry-color: #3858e9;
 	}
 
 	.chat-feedback-wrapper {
+		
+		$foldable-header-font-size: 0.75rem;
```

Comment:
I hoped we could use some sort of variable like `font-size: $font-...`. No need to declare a variable inside of here that is going to be used a single time!